sdk/keyvault/keyvault-keys/src/cryptographyClientModels.ts

```diff
@@ -7,18 +7,48 @@ import type { JsonWebKey } from "./generated/models/index.js";
 import {
   JsonWebKeyEncryptionAlgorithm as EncryptionAlgorithm,
   JsonWebKeyCurveName as KeyCurveName,
-  KnownJsonWebKeyCurveName as KnownKeyCurveNames,
-  KnownJsonWebKeySignatureAlgorithm as KnownSignatureAlgorithms,
   JsonWebKeySignatureAlgorithm as SignatureAlgorithm,
 } from "./generated/models/index.js";
 
-export {
-  KeyCurveName,
-  KnownKeyCurveNames,
-  EncryptionAlgorithm,
-  SignatureAlgorithm,
-  KnownSignatureAlgorithms,
-};
+export { KeyCurveName, EncryptionAlgorithm, SignatureAlgorithm };
+
+/** Known values of {@link KeyCurveName} that the service accepts. */
+export enum KnownKeyCurveNames {
+  /** The NIST P-256 elliptic curve, AKA SECG curve SECP256R1. */
+  P256 = "P-256",
+  /** The NIST P-384 elliptic curve, AKA SECG curve SECP384R1. */
+  P384 = "P-384",
+  /** The NIST P-521 elliptic curve, AKA SECG curve SECP521R1. */
+  P521 = "P-521",
+  /** The SECG SECP256K1 elliptic curve. */
+  P256K = "P-256K",
```

Comment:
Generated incorrectly as P256_K

Reply:
Cc @MaryGao -- this looks like another example of the enum normalization being different in Modular vs HLC (https://github.com/Azure/autorest.typescript/issues/2832)

Reply:
Yes, this is an accepted breaking [discussed here](https://github.com/Azure/autorest.typescript/issues/2832#issuecomment-2499507467), if you think the name here is not reasonable, you could use client name to rename.

```
@@clientName(JsonWebKeyCurveName.P256_K, "P256K", "javascript");
```